source/documentation/annexes.md

```diff
@@ -26,16 +26,16 @@ Current memory limits:
 <div style="height:0px;font-size:0px;">&nbsp;</div>
 | Container type | Requested | Limit    |
 | -------------- | --------- | -------- |
-| R Studio | 8 GB (was 5 GB) | 20 GB |
-| Jupyter | 1 GB | 12 GB |
-| App | no limit | no limit |
+| R Studio | 12 GB | 12 GB |
+| Jupyter | 12 GB | 12 GB |
+| App | 100 MB | 8 GB |
```

Comment:
@rossjones , I am kind of puzzled about the figures for App, there is no. limit defined in our helm chart, but I checked the pod in our existing alpha cluster, they do have the figures you put above,  not sue how this figures `100 MB | 8 GB ` come from

Reply:
```
kubectl describe pod statistics-explorer-webapp-5fbf4b9cd7-9wlhb -n apps-prod
.
.
  webapp:
    Limits:
      cpu:     2
      memory:  8Gi
    Requests:
      cpu:      10m
      memory:   100Mi
.
```